src/coreclr/jit/gentree.cpp

```diff
@@ -8603,6 +8603,34 @@ GenTreeBlk* Compiler::gtNewStoreBlkNode(ClassLayout* layout, GenTree* addr, GenT
     return store;
 }
 
+//------------------------------------------------------------------------------
+// gtNewStoreDynBlkNode : Create a dynamic block store node.
+//
+// Arguments:
+//    addr        - Destination address
+//    data        - Value to store (init val or indirection representing a location)
+//    dynamicSize - Node that computes number of bytes to store
+//    indirFlags  - Indirection flags
+//
+// Return Value:
+//    The created GT_STORE_DYN_BLK node.
+//
+GenTreeStoreDynBlk* Compiler::gtNewStoreDynBlkNode(GenTree*     addr,
+                                                   GenTree*     data,
+                                                   GenTree*     dynamicSize,
+                                                   GenTreeFlags indirFlags)
+{
+    assert((indirFlags & GTF_IND_INVARIANT) == 0);
+    assert(data->IsInitVal() || data->OperIs(GT_IND));
+
+    GenTreeStoreDynBlk* store = new (this, GT_STORE_DYN_BLK) GenTreeStoreDynBlk(addr, data, dynamicSize);
+    store->gtFlags |= GTF_ASG;
```

Comment:
nit: it's already set in GenTreeStoreDynBlk ctror

Reply:
I removed that in this PR (just to make it consistent with `gtNewStoreBlkNode`)

Reply:
ah, I am blind